conans/test/functional/toolchains/cmake/cmakedeps/test_cmakedeps_aliases.py

```diff
@@ -148,7 +148,7 @@ def package_info(self):
     client.save({"conanfile.py": consumer, "CMakeLists.txt": cmakelists})
     client.run("create .")
 
-    assert "Target name 'hello::hello' already exists." in client.out
+    assert "Target name 'hello::hello' already exists." not in client.out
```

Comment:
I was doubting if keeping these tests or not... I think it could be helpful to have them. Wdyt?